config/typescript.js

```diff
@@ -126,6 +150,10 @@ module.exports = {
     '@typescript-eslint/quotes': ['error', 'single', { 'avoidEscape': true, 'allowTemplateLiterals': true }],
     'quotes': 'off',
 
+    '@typescript-eslint/require-await': 'off', // Requires type information
```

Comment:
Probably prudent to disable the base rules for these too eh? Otherwise it might crash or misbehave in some manner

Reply:
Eh, for the rules that _are_ enabled, disabling the core rule reduces the errors that show up (i.e. 1 error on the line vs. 2) but there are cases where false positives could exist. These two are fine.

Reply:
> These two are fine.

We have `return-await` disabled in the core rules and false positives shouldn't be a concern for `require-await`.

Reply:
OK. I just mentioned it because of this warning in the docs:
>In some cases, ESLint provides a rule itself, but it doesn't support TypeScript syntax; either it crashes, or it ignores the syntax, or it falsely reports against it.

If you've investigated the rule specifically, I guess we can leave it on 🤷‍♂

Reply:
It seems to work on the two projects I've tested it with. We can disable it in a patch release if need be.